lib/hiera/interpolate.rb

```diff
@@ -9,13 +9,14 @@ class << self
     INTERPOLATION = /%\{([^\}]*)\}/
     METHOD_INTERPOLATION = /%\{(scope|hiera|literal|alias)\(['"]([^"']*)["']\)\}/
 
-    def interpolate(data, scope, extra_data, recurse_guard)
+    def interpolate(data, scope, extra_data, options={:recurse_guard => nil, :order_override => nil})
```

Comment:
This is the only area where I wasn't sure if we needed to keep backwards compatibility. It looks like we've already decided not to, so I can drop the use of a hash here and require all arguments.